packages/integration-sdk-runtime/src/synchronization/index.ts

```diff
@@ -136,9 +140,23 @@ function buildJobConfiguration({
   integrationInstanceId,
   integrationJobId,
 }: SynchronizeInput) {
-  return source === 'api'
-    ? { source, scope }
-    : { source, integrationInstanceId, integrationJobId };
+  if (source === 'api') {
+    return {
+      source,
+      scope,
+    };
+  }
+
+  if (source === 'system-internal') {
+    return {
+      source,
+      scope,
```

Comment:
We need to pass scope through for this source

Reply:
Currently the SDK is public facing. We don't want external users incorrectly setting the source. 🤔

Reply:
They would have to intentionally set their source as `system-internal` which would be a case of self sabotage. I agree that ideally they would not be able to use this but I am not sure we have an alternative as internal integrations use the same SDK.